src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,22 @@
+const ApiError = require('../exeptions/api.error.js');
+
+const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+  }
```

Comment:
Add a `return;` statement here to prevent the execution of the subsequent code block when the error is an instance of `ApiError`.